src/platform/monitoring/frontend-metrics/feature-flag.js

```diff
@@ -4,7 +4,5 @@
  * @module platform/monitoring/frontend-metrics/feature-flag
  */
 export default function isMetricsEnabled() {
-  const environments = ['vagovdev', 'vagovstaging', 'vagovprod'];
-
-  return !!environments.includes(__BUILDTYPE__);
+  return true;
 }
```

Comment:
This should also be torn down entirely, unless for some reason we want the feature flag to solely act as a method of excluding local builds.